internal/hcsoci/create.go

```diff
@@ -165,33 +165,30 @@ func CreateContainer(ctx context.Context, createOptions *CreateOptions) (_ cow.C
 	}
 
 	var hcsDocument, gcsDocument interface{}
-	log.G(ctx).Debug("hcsshim::CreateContainer allocating resources")
 	if coi.Spec.Linux != nil {
 		if schemaversion.IsV10(coi.actualSchemaVersion) {
 			return nil, resources, errors.New("LCOW v1 not supported")
 		}
-		log.G(ctx).Debug("hcsshim::CreateContainer allocateLinuxResources")
+		log.G(ctx).Debug("allocating LCOW resources")
```

Comment:
I feel like the **hcsshim::FunctionName** prefix should stay for these. What's wrong with **hcsshim::internal/hcsoci.CreateContainer - allocating LCOW resources** and the same for the others? You can obviously just search for the debug/err message in the codebase and find where it was emitted but I feel its much easier to just have the exact package and function call the message originated from as well.

Reply:
Ehhh I guess typed out that does look kind of unwieldy.

Reply:
I don't think the function name adds much at all.

- We have so many similarly named functions spread around, knowing the name doesn't tell you exactly what it's doing.
- I would rather the messages be more readable, rather than having a bunch of other junk in them.
- We already log the span context, which is another similar way of recording execution state.

Reply:
And it's easy to search for the log message to find where it comes from.

Reply:
Yea I'm starting to agree, for our purposes I think concise might be better. The dir/package.FunctionCall solves the which of the 20 different CreateContainers is this search, but I feel its got too much junk in it by then like you said